Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -1455,10 +1464,11 @@ def create_test_file(is_nightly, skip_save=False, path_to_pack=''):
     parser.add_argument('-s', '--skip-save', type=tools.str2bool,
                         help='Skipping saving the test filter file (good for simply doing validation)')
     parser.add_argument('-p', '--changed_pack_path', type=str, help='A string representing the changed files')
+    parser.add_argument('-t', '--marketplace_version', type=str, help='Is xsoar or marketplacev2')
```

Comment:
Let's try to be consistent with the rest of the build that uses the `--marketplace` convention
```suggestion
    parser.add_argument('--marketplace', type=str, help='Is xsoar or marketplacev2')
```